WordPress/Classes/ViewRelated/Blog/Blog Details/BlogDetailsViewController+Me.swift

```diff
@@ -9,14 +9,15 @@ extension BlogDetailsViewController {
             return
         }
 
-        ImageDownloader.shared.downloadGravatarImage(with: email) { [weak self] image in
-            guard let image,
-                  let gravatarIcon = image.gravatarIcon(size: Metrics.iconSize) else {
-                return
+        Task { @MainActor [weak self] in
+            do {
+                let service = Gravatar.AvatarService()
+                let result = try await service.fetch(with: .email(email))
```

Comment:
I recommend always passing the preferred size otherwise the server will default to  80px, which can be working fine in some cases but it is usually a coincidence.

AFAIU we want the size `Metrics.iconSize` in points, so we can call this like:

```
service.fetch(with: .email(email), options: .init(preferredSize: .points(Metrics.iconSize)))
```